smartcontract/service/native/shardmgmt/shardmgmt.go

```diff
@@ -116,6 +117,8 @@ func ShardMgmtInit(native *native.NativeService) ([]byte, error) {
 			ShardID:             native.ShardID,
 			GenesisParentHeight: native.Height,
 			State:               shardstates.SHARD_STATE_ACTIVE,
+			Config:              &shardstates.ShardConfig{VbftConfigData: &config.VBFTConfig{}},
```

Comment:
an empty shard config?
why not nil?

Reply:
in case of nil pointer error while serialize